types/google.analytics/index.d.ts

```diff
@@ -524,7 +525,7 @@ declare namespace UniversalAnalytics {
         socialNetwork?: string;
         socialTarget?: string;
         some?: string;
-        step?: boolean;
+        step?: number;
```

Comment:
You are right, I have made it either or. I have also now gone through and updated all metrics to also be either string or number based on https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets